src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -62,9 +61,9 @@ private void InitRadioButtonsDescription()
             RadioMediumDesc.Inlines.Add(new Run(Res.ChangeScaleFactorPromptDescriptionDefaultSetting) { FontWeight = FontWeights.Bold });
             RadioMediumDesc.Inlines.Add(" " + viewModel.OptionsGeometryScale.DescriptionScaleRange[1]);
 
-            RadioLargeDesc.Inlines.Add(viewModel.OptionsGeometryScale.DescriptionScaleRange[1]);
+            RadioLargeDesc.Inlines.Add(viewModel.OptionsGeometryScale.DescriptionScaleRange[2]);
```

Comment:
geometry scaling descriptions were repeated for medium and large, and extra large was wrong.